unfoldingWord/translationCore
# bugfix 3273/UGNT QA Fail

#### This pull request addresses:

FIx for greek drag and drop.


#### How to test this pull request:

Greek drag and drop should work now to combine into phrases and split phrases.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3674)
